src/containers/Admin/EditTab/BikePanel/index.tsx

```diff
@@ -59,19 +59,21 @@ function BikePanel(props: Props): JSX.Element {
                 <Checkbox
                     key={id}
                     id={id}
-                    name={name}
+                    name={getTranslation(name) || id}
```

Comment:
Har dere innført getTranslation? Kanskje funksjonen kan returnere id-en dersom feltet ikke eksisterer, så slipper dere å manuelt sjekke hver gang den brukes? :)

Reply:
Hm, eg synst det er fint at `getTranslation` er ein util som berre henter ut riktig tekst basert på ein MultilingualString, `id` er på ein måte ikkje knytta til det. 

Kvifor blir det defaulta til ID her egentlig? Det er vel typ `YTI:Station:123345567` og rart å bruke som name? 🤔

Reply:
⏫  Høres ut som den beste løsningen om det er mulig å oppnå. BFFen garanterer vel at nb, nn og en returneres? Kanskje ulempen med at APIet returnerer et array og ikke et objekt, som gjør det litt vanskeligere å håndtere klienten, ref:
```
export function getTranslation(
    translationObject: TranslatedString,
    languageId = 'nb',
): string | null {
    const translations: Translation[] = translationObject.translation
    const match = translations.find(
        (currentTranslation) => currentTranslation.language === languageId,
    )
    if (!match) return null
    return match.value
}
```

Reply:
Ja, `getTranslation` blir innført med denne PR'en. Det er tenkt at den skal være mer generell og derfor håndterer den ikke et feilet treff selv (ved f.eks. å returnere ID). 
ID er bare en fallback-mekanisme her som du er inne på, tenkte at det er bedre enn f.eks. å defaulte til tom streng - da vil man i hvert fall ha en synlig utfylt komponent (hvis noe skulle være galt), men hvis noen har et bedre forslag så er jeg veldig åpen for det:)

Reply:
Ser jo forøvrig at jeg fallbacker til tom streng i BikeTile, så det blir jo veldig inkonsistent 🙈

Tror ikke nb, nn og en er garantert, nb var i hvert fall den eneste verdien jeg fikk tak i mens jeg testet. Men som du sier så blir det litt tricky pga. array-formatet. Et objekt med en enum for de ulike språkene hadde kanskje vært lettere å forholde seg til på klientsiden, men blir kanskje mer omfattende å organisere backend.

Reply:
Fikset defaults til tom streng `''` så det blir konsistent.

@draperunner Mente du at du heller vil ha typesignaturen i funksjonen sånn her ?
``` javascript
export function getTranslation(
    translationObject: TranslatedString,
    languageId: MultilingualString['language'] = 'nb',
): string | null 
```

Reply:
Sorry, eg blanda MultilingualString og TranslatedString trur eg. TranslatedString er det riktige frå Mobility-APIet.